### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by eclee25**
For manuscript submission

### CODE REVIEW

**Comment by eclee25** [comment]
Instead of merging eclee25-patch-1 which wasn't passing unit tests back into master, I made the changes to the README file in this branch instead. Not sure why the deploy tests aren't passing though.

File: .github/workflows/aws-apl.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# This workflow will build and push a new container image to Amazon ECR The following information will need to be included in your Github secrets: - AWS_ACCESS_KEY_ID: dedicated
```
Comment (jkamins7) [inline .github/workflows/aws-apl.yaml:1]: is this working now?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -16,11 +16,17 @@ jobs:
   unit-tests:
     runs-on: ubuntu-18.04
     container:
-      image: docker://hopkinsidd/covidscenariopipeline:latest-dataseed
+      image: docker://hopkinsidd/covidscenariopipeline:latest-dev
```
Comment (jkamins7) [inline .github/workflows/ci.yml:19]: this should be latest i guess

Comment (eclee25) [reply .github/workflows/ci.yml]: Changed this to latest

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -16,11 +16,17 @@ jobs:
   unit-tests:
     runs-on: ubuntu-18.04
     container:
-      image: docker://hopkinsidd/covidscenariopipeline:latest-dataseed
+      image: docker://hopkinsidd/covidscenariopipeline:latest
```
Comment (jkamins7) [inline .github/workflows/ci.yml:19]: you can change this back to dev i guess

File: Dockerfile [hunk 4]
```diff
@@ -12,6 +12,7 @@ ENV LC_ALL en_US.UTF-8
 # set noninteractive installation
 ENV DEBIAN_FRONTEND noninteractive
 ENV R_VERSION 3.6.3-1bionic
+ENV RSTUDIO_VERSION 1.2.5033
```
Comment (jkamins7) [inline Dockerfile:15]: why is there an rstudio version in the dockerfile now?

File: Dockerfile [hunk 5]
```diff
@@ -68,6 +70,7 @@ RUN apt-get update && \
     supervisor \
     awscli \
     r-base-dev=$R_VERSION \
+    openjdk-8-jdk \
```
Comment (jkamins7) [inline Dockerfile:73]: similarly, why is there java in the dockerfile?

File: Dockerfile [hunk 6]
```diff
@@ -87,14 +90,20 @@ ENV HOME /home/app
...
-
+RUN Rscript -e "install.packages('packrat',repos='https://cloud.r-project.org/')" \
+    && Rscript -e "install.packages('arrow',repos='https://cloud.r-project.org/')" \
+    && Rscript -e 'arrow::install_arrow()'
+COPY --chown=app:app packrat $HOME/packrat
+COPY --chown=app:app Docker.Rprofile $HOME/.Rprofile
+COPY --chown=app:app R/pkgs $HOME/R/pkgs
+RUN Rscript -e 'packrat::restore()' \
+    && curl -O https://download2.rstudio.org/server/bionic/amd64/rstudio-server-$RSTUDIO_VERSION-amd64.deb \
+    && sudo apt-get install -f -y ./rstudio-server-$RSTUDIO_VERSION-amd64.deb \
```
Comment (jkamins7) [inline Dockerfile:101]: more rstudio in the docker

File: Dockerfile [hunk 7]
```diff
@@ -87,14 +90,20 @@ ENV HOME /home/app
...
+    && rm -f ./rstudio-server-$RSTUDIO_VERSION-amd64.deb
+RUN Rscript -e 'install.packages(list.files("R/pkgs",full.names=TRUE),type="source",repos=NULL)' \
+
+# expose Rstudio port
+EXPOSE 8787
```
Comment (jkamins7) [inline Dockerfile:106]: more rstudio stuff

File: R/pkgs/report.generation/NAMESPACE [hunk 8]
```diff
@@ -24,6 +24,7 @@ export(make_scn_time_summary_table)
 export(make_scn_time_summary_table_withVent)
```
Comment (jkamins7) [inline R/pkgs/report.generation/NAMESPACE:24]: Wait, why are we moving from report_generation to report.generation?  Confused....

File: packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION [hunk 9]
```diff
@@ -0,0 +1,36 @@
+Package: covidImportation
+Title: Estimating the Number of Imported COVID-19 Cases
+Version: 1.5
```
Comment (jkamins7) [inline packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION:3]: hm... this looks wrong

File: R/pkgs/report.generation/NAMESPACE (hunk 8 above)
Comment (eclee25) [reply R/pkgs/report.generation/NAMESPACE:24]: report.generation is the correct, good name. Same as what we have in `inference`

File: vignettes/how_to.Rmd [hunk 10]
```diff
@@ -491,7 +491,7 @@ Severe:
  - Run simulate.py
```
Comment (jkamins7) [inline vignettes/how_to.Rmd:491]: Is this file still useful?

File: packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION (hunk 9 above)
Comment (jkamins7) [reply packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION:3]: The importation tests are passing, so I guess this version is ok.

Comment (shauntruelove) [reply packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION:3]: can we change the importation package to the tag "v1.8"? I have updated to fix some errors that might have been causing issues.

**Review by shauntruelove — DISMISSED** [review]
updated covidImportation to fix issues caused by what seems to have been changes in some of the packages. v1.8 is now the current, stable release.

@jkamins7 is changing the package version in CSP

**Review by jkamins7 — APPROVED** [review]

**Comment by shauntruelove** [comment]
Looks good.

**Review by shauntruelove — APPROVED** [review]

**Pull request merged**